libs/vertexai/langchain_google_vertexai/chat_models.py

```diff
@@ -614,6 +615,23 @@ def validate_environment(cls, values: Dict) -> Dict:
     def _is_gemini_advanced(self) -> bool:
         return self.model_family == GoogleModelFamily.GEMINI_ADVANCED
 
+    def _get_ls_params(
+        self, stop: Optional[List[str]] = None, **kwargs: Any
+    ) -> LangSmithParams:
+        """Get standard params for tracing."""
+        params = self._get_invocation_params(stop=stop, **kwargs)
```

Comment:
maybe use `self._prepare_params` instead to respect kwargs (there's some mapping of arg names happening there)? Or probably we should rather redefine `_getinvocation_params` method for Vertex integrations :).

https://github.com/langchain-ai/langchain-google/blob/2f7be6f1d6b80fab0b2059a1265c0114120013fe/libs/vertexai/langchain_google_vertexai/_base.py#L220

Reply:
thanks, made this change.